pdfmaker/css/holt/pdf.css

```diff
@@ -1,9 +1,1175 @@
+  margin-bottom: 18pt; }    
+
+p[class*="AboutAuthorText"] {
+  padding: 0 24pt 0 24pt;
+  font-size: 10.5pt;
+  line-height: 18pt; }
+
+h1 + p[class*="AboutAuthorText"],
+figure + p[class*="AboutAuthorText"],
+p.AboutAuthorTextNo-Indentatatx1,
+p.IllustrationSourceis + p[class*="AboutAuthorText"] {
+  text-indent: 0; }
+
+section h1.AboutAuthorTextHeadatah + figure + p, section.abouttheauthor h1 + figure + p {
+  margin-top: 22pt; }
+
+section[data-type="preface"] *[class*="FMSubheadfmsh"]:first-of-type {
+  margin-top: -130.65pt; } 
+
+section[data-type="chapter"] {
+  page: chapter;
+  page-break-before: always;
+  prince-page-group: start;
+  counter-reset: footnote; }
+
+section[data-type="chapter"] ~ section[data-type="chapter"], {
+   counter-reset: none }
+
+h1[class*="ChapTitle"]:first-child, 
+h1[class*="ChapNumber"]:first-child {
+  margin-top: 70pt; 
+  padding-bottom: 0;
+  margin-bottom: 0; }  
+
+h1[class*="ChapTitle"]:not(:first-child) {
+  margin-top: 0; }   
+
+section[data-type="preface"] + section[data-type="chapter"], 
+section[data-type="foreword"] + section[data-type="chapter"], 
+section[data-type="titlepage"] + section[data-type="chapter"], 
+section[data-type="halftitlepage"] + section[data-type="chapter"], 
+section[data-type="copyright-page"] + section[data-type="chapter"], 
+section[data-type="dedication"] + section[data-type="chapter"],
+nav[data-type="toc"] + section[data-type="chapter"] {
+  counter-reset: page 1 footnote;
+  page-break-before: right; }
+
+section[data-type="preface"] + div[data-type="part"], 
+section[data-type="foreword"] + div[data-type="part"], 
+section[data-type="titlepage"] + div[data-type="part"], 
+section[data-type="halftitlepage"] + div[data-type="part"], 
+section[data-type="copyright-page"] + div[data-type="part"], 
+section[data-type="dedication"] + div[data-type="part"],
+nav[data-type="toc"] + div[data-type="part"] {
+  counter-reset: page 1; }
+
+div[data-type="part"] + section[data-type="chapter"] {
+  page-break-before: right; }
+
+section[data-type="chapter"] h1[class*="ChapNumbercn"] {
+  font-size: 20pt;
+  padding-top: 3pt;
+  line-height: 15pt;
+  font-weight: normal;
+  text-align: center;
+  margin-bottom: 10pt;
+  text-indent: 0; }
+
+section[data-type="chapter"] h1[class*="ChapTitleNonprintingctnp"] {
+  counter-increment: chapternumber; 
+  string-set: section content(); }
+
+/*chapter title!*/
+section[data-type="chapter"] h1[class*="ChapTitlect"] {
+  font-size: 15pt;
+  line-height: 19pt;
+  font-weight: normal;
+  text-align: center;
+  padding-left: 39pt;
+  padding-right: 39pt;
+  margin-bottom: 9pt;
+  counter-increment: chapternumber; 
+  string-set: section content(); }
+
+h1[class*="ChapTitlect"][data-autolabel="yes"]:before {
+  content: attr(data-labeltext);
+  display: block;
+  padding-top: 3pt;
+  font-size: 20pt;
+  line-height: 15pt;
+  font-weight: normal;
+  text-align: center;
+  margin-bottom: 10pt;
+  text-indent: 0; }
+
+[class*="Nonprinting"] {
```

Comment:
add a * at the beginning of this def: \*[class*="Nonprinting"]. (It's totally possible this is a typo in my code as well...)